include/circt/Dialect/SV/SV.td

```diff
@@ -26,8 +26,12 @@ def SVDialect : Dialect {
     This dialect defines the `sv` dialect, which represents various
     SystemVerilog-specific constructs in an AST-like representation.
   }];
-
+  let dependentDialects = ["circt::comb::CombDialect"];
```

Comment:
Why does this need to be added? It seems to be the only dialect which has it added.

Reply:
Its burried in the changes here, but the old constructor was explicitly loading this dialect:
```c++
SVDialect::SVDialect(MLIRContext *context)
    : Dialect(getDialectNamespace(), context,
              ::mlir::TypeID::get<SVDialect>()) {
  context->loadDialect<circt::comb::CombDialect>();
   ...
  ```
 Adding this line to ODS generates this in the new constructor:
 ```c++
   explicit SVDialect(::mlir::MLIRContext *context)
    : ::mlir::Dialect(getDialectNamespace(), context,
      ::mlir::TypeID::get<SVDialect>()) {
    
    getContext()->getOrLoadDialect<circt::comb::CombDialect>();

    initialize();
  }
  ```

Reply:
Looks like [it was originally added ](https://github.com/llvm/circt/commit/4d48b77be0620191da0a4aae002f392d0f9870a4#diff-737aadd546bc48dd112bdf61bcf5f3a1fe7603a2c4087324e304027033209f74L28)to be able to use Comb dialect ops in an SV op canonicalization, I don't really know if there is a better way to handle it but it looks ok to me (maybe cross-dialect canonicalizations should go in a different place?).